packages/xstate-wallet/src/utils/workflow-utils.ts

```diff
@@ -72,3 +83,53 @@ export const debugAction = (c, e, {state}) => {
   // eslint-disable-next-line no-debugger
   debugger;
 };
+
+export const displayUI = (
+  messagingService: MessagingServiceInterface
+): ActionFunction<any, any> => () => messagingService.sendDisplayMessage('Show');
```

Comment:
There's a genuine change here from the code that it replaced:
```ts
const displayUI = (messagingService: MessagingServiceInterface): ActionObject<Context, Event> => ({
  type: 'displayUI',
  exec: () => messagingService.sendDisplayMessage('Show')
});
```
The original code includes a `type`, which is what is displayed on the xstate viewer, if you display the machine directly. Would be good to keep that property. (Unless there's something going on here that I don't understand. Did you manage to get your machine to display in xstate devtools?)

Reply:
Is `xstate-devtools` the visualizer? The `hideUI/displayUI` seem to show ok up https://xstate.js.org/viz/?gist=8d968ac87a9dd068c51f1c6c8ca8f9bf